plugins/github/main.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"math"
+	"net/http"
+	"os"
+
+	"github.com/alecthomas/jsonschema"
+	"github.com/falcosecurity/plugin-sdk-go/pkg/sdk"
+	"github.com/falcosecurity/plugin-sdk-go/pkg/sdk/plugins"
+	"github.com/falcosecurity/plugin-sdk-go/pkg/sdk/plugins/extractor"
+	"github.com/falcosecurity/plugin-sdk-go/pkg/sdk/plugins/source"
+	"github.com/falcosecurity/plugin-sdk-go/pkg/sdk/symbols/extract"
+	"github.com/google/go-github/github"
+	"github.com/valyala/fastjson"
+	"golang.org/x/oauth2"
+)
+
+// Plugin info required by the framework.
+const (
+	PluginRequiredApiVersion        = "0.2.0"
+	PluginID                 uint32 = 666
```

Comment:
This should get its own ID and be added to the registry! I think the next in line is ID=8!